src/main/telemetry/msp_shared.c

```diff
@@ -101,7 +98,7 @@ bool handleMspFrame(uint8_t *frameStart, uint8_t *frameEnd)
     }
 
     mspPacket_t *packet = mspPackage.requestPacket;
-    sbuf_t *frameBuf = sbufInit(&mspPackage.requestFrame, frameStart, frameEnd);
+    sbuf_t *frameBuf = sbufInit(&mspPackage.requestFrame, frameStart, (uint8_t)frameLength+frameStart);
```

Comment:
'frameStart + frameLength' (add offset to pointer, not other way around)

Reply:
Done.